include/boost/geometry/algorithms/union.hpp

```diff
@@ -209,7 +347,8 @@ inline OutputIterator union_insert(Geometry1 const& geometry1,
 {
     concepts::check<Geometry1 const>();
     concepts::check<Geometry2 const>();
-    concepts::check<GeometryOut>();
+    //concepts::check<GeometryOut>();
```

Comment:
This line is replaced and I think you can delete it?

Reply:
Done.